package.json

```diff
@@ -14,16 +14,17 @@
     "esri-dump": "cli.js"
   },
   "main": "index.js",
-  "license": "BSD",
+  "license": "MIT",
```

Comment:
npm has been complaining that this is not a standard license name, I assumed that since the license in the repo is MIT the package.json should reflect this as well. If we are dual licensing I'll happily switch it back as well as include a copy of the BSD license in the repo.